README.md

```diff
@@ -112,6 +115,8 @@ module.exports = {
     env: {
       ROOT_URL: 'http://app.com',
       MONGO_URL: 'mongodb://localhost/meteor'
+      PORT: 80, // (optional) defaults to 80
+      BIND: '0.0.0.0' // (optional) defaults to 0.0.0.0
```

Comment:
Removed `PORT` and `BIND` from `env`.

 `PORT` is in meteor root object, and I corrected `modules/meteor/index.js` according to variable position. 

`BIND` is in `docker`  object now.